pygbm/loss.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+# TODO: ABC for losses, regression losses and classification losses
+class LeastSquares:
+    def __init__(self):
+        self.hessians = np.full(shape=1, fill_value=2, dtype=np.float32)
```

Comment:
Please make that attribute private: `self._constant_hessian = np.full(shape=1, fill_value=2, dtype=np.float32)`.